vllm/_custom_ops.py

```diff
@@ -190,17 +190,25 @@ def gptq_marlin_24_gemm(a: torch.Tensor, b_q_weight: torch.Tensor,
 
 
 # cutlass
-def cutlass_scaled_mm_dq(a: torch.Tensor, b: torch.Tensor,
-                         scale_a: torch.Tensor, scale_b: torch.Tensor,
-                         out_dtype: Type[torch.dtype]) -> torch.Tensor:
+def cutlass_scaled_mm_dq(
+        a: torch.Tensor,
+        b: torch.Tensor,
+        scale_a: torch.Tensor,
+        scale_b: torch.Tensor,
+        out_dtype: Type[torch.dtype],
+        bias: Optional[Type[torch.dtype]] = None) -> torch.Tensor:
```

Comment:
Shouldn't this be `Optional[torch.Tensor]`?

Reply:
fixed